pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
@DanKerns, the operator does not have any functionality available besides creating/configuring a cluster.  We need to discuss the extent to which we use more TPRs with the operator for other functionality in the future, but for the time being we need to use the existing Rook API.